Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN curl -L ${FLYWAY_ARTIFACT_URL}${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_
   && chmod a+x /flyway/flyway
 
 ENV PATH="/flyway:${PATH}"
+ENV REDGATE_DOCKER=true
```

Comment:
This section is for the OSS Flyway docker image. Please move this down to below line 21 for the Redgate image

Reply:
Thanks - Done in 16fb418